pallets/frequency-tx-payment/src/capacity_stable_weights.rs

```diff
@@ -51,7 +51,7 @@ use sp_std::marker::PhantomData;
 /// - 95th: 91_299
 /// - 75th: 90_626
 pub const CAPACITY_EXTRINSIC_BASE_WEIGHT: Weight =
-	Weight::from_ref_time(WEIGHT_REF_TIME_PER_NANOS.saturating_mul(90_764));
+	Weight::from_parts(WEIGHT_REF_TIME_PER_NANOS.saturating_mul(90_764), 0);
```

Comment:
None of the values here were changed, `from_ref_time()` was deprecated and fixed. 
Note the second argument is the proof size, and setting to 0 was how Substrate handled the deprecation.

@enddynayn We may want to evaluate updating this file against the weights generated in this PR.